README.md

```diff
@@ -8,15 +8,31 @@
 # aries-framework-go
 
 ## Introduction
-A go framework for Aries
+A Go framework for Aries
 
-## Contributing
+## Build
+##### Prerequisites (General)
+- Go 1.12
 
-Found a bug? Ready to submit a PR? Want to submit a proposal for your grand
-idea? Follow our [guidelines](.github/CONTRIBUTING.md) for more information
-to get you started!
+##### Prerequisites (for running tests)
+- Go 1.12
+- Docker
+- Make
+- OpenSSL
```

Comment:
We're using docker image `frapsoft/openssl` by default. The developer may override this by setting the `OPENSSL_CMD` enviroment variable.

Reply:
[Removed](https://github.com/hyperledger/aries-framework-go/compare/bc1bec9caa24a8ee97ec09d6762fe66a2e5bdefb..1c758ac9c8a5372da72239e742bd1fd802b7b515).